extensions/wikia/PhalanxII/tests/PhalanxServiceTest.php

```diff
@@ -22,7 +22,7 @@ public function checkPhalanxAlive( ) {
 		$this->service = new PhalanxService();
 		if (!$this->service->status()) {
 			//Skip test if phalanx service is not available
-			throw new Exception("Can't connect to phalanx service on " . $this->app->wg->PhalanxServiceUrl);
+			throw new Exception("Can't connect to phalanx service");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS112)